packages/@aws-cdk/aws-pipes-targets-alpha/test/integ.lambda.handler/handler.ts

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { LambdaClient, TagResourceCommand } from '@aws-sdk/client-lambda';
+
+exports.handler = async (event: any, context: any) => {
+  const client = new LambdaClient();
+  // eslint-disable-next-line no-console
+  console.log('Received event:', JSON.stringify(event, null, 2));
```

Comment:
Do we need the log statements? Or ist a leftover from debugging.

Reply:
leftover from debugging :D